experimental/packages/opentelemetry-instrumentation-fetch/src/fetch.ts

```diff
@@ -58,26 +60,37 @@ export interface FetchCustomAttributeFunction {
  * FetchPlugin Config
  */
 export interface FetchInstrumentationConfig extends InstrumentationConfig {
-  // the number of timing resources is limited, after the limit
-  // (chrome 250, safari 150) the information is not collected anymore
-  // the only way to prevent that is to regularly clean the resources
-  // whenever it is possible, this is needed only when PerformanceObserver
-  // is not available
+  /** Function for adding custom attributes on the span */
+  applyCustomAttributesOnSpan?: FetchCustomAttributeFunction;
+
+  /**  the number of timing resources is limited, after the limit
+  (chrome 250, safari 150) the information is not collected anymore
+  the only way to prevent that is to regularly clean the resources
+  whenever it is possible, this is needed only when PerformanceObserver
+  is not available */
   clearTimingResources?: boolean;
-  // urls which should include trace headers when origin doesn't match
-  propagateTraceHeaderCorsUrls?: web.PropagateTraceHeaderCorsUrls;
+
+  /** List of request headers to include as attributes on the span. */
+  requestHeadersAsAttributes?: string[];
```

Comment:
Q: is it on the roadmap to add `requestHeadersAsAttributes`/`responseHeadersAsAttributes` to Xhr instrumentation?